### PR TITLE
Add documentation for SAML SSO on default projects and Okta

### PR DESCRIPTION
**Pull request by mostafa**
Changes in this PR:
1. Add documentation about SAML SSO setup on Okta.
2. Add a separate SAML SSO page for the introduction, supported IdPs and the new SAML SSO role mappings to set default project(s).
3. Replace Azure AD SAML SSO (`/cloud/project-and-team-management/azure-ad-saml-sso/`) to Azure AD (`/cloud/project-and-team-management/saml-sso/azure-ad/`).

### CODE REVIEW

File: src/data/markdown/docs/03 cloud/05 Project and Team Management/04 SAML SSO.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+To setup SAML SSO for your organization, some requirements should be satisfied and some steps should be taken on your IdP and on k6 Cloud. Assuming you already have admin access to your IdP and the k6 Cloud, this is the usual process to enable SAML SSO for your organization on k6 Cloud:
+
+1. Install or setup the SAML SSO application on your IdP.
+2. Provide the "IdP Metadata URL" to the customer success team.
+3. Update the application on the IdP with the provided token from the customer success team.
+4. Assign users to the IdP application.
+5. Optionally choose default projects for newly provisioned users to land in on k6 Cloud.
+
+## Supported IdPs
```
Comment (Vmihajlovic) [inline src/data/markdown/docs/03 cloud/05 Project and Team Management/04 SAML SSO.md:26]: wasn't it mentioned at some point that basically any provider would work as long as they use SAML 2.0 standard? If that is so, perhaps we can add a line here somewhere to reflect that?

File: src/data/markdown/docs/03 cloud/05 Project and Team Management/04 SAML SSO.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+1. Make sure you correctly setup the SAML organization with the help of the customer success team. You can use your admin account to test the setup.
+2. Go to your organization setting and choose the SAML SSO tab:
+![SAML SSO settings](images/04-SAML-SSO/saml-sso-settings.png)
+As you can see below, the default project is already selected:
+![SAML SSO default project](images/04-SAML-SSO/saml-sso-default-project.png)
+3. From the drop-down you can choose the default project(s) newly provisioned users will be assigned, then click on update button:
+![SAML SSO update default projects](images/04-SAML-SSO/saml-sso-update-default-project.png)
+4. Now the users assigned to the IdP application can sign in to k6 Cloud and they'll be automatically added to the specified project(s) in step 3 with __project read/write__ permissions.
+
```
Comment (Vmihajlovic) [inline src/data/markdown/docs/03 cloud/05 Project and Team Management/04 SAML SSO.md:49]: we also might want to mention enforce SAML login feature or basically just point out that we can have the ability to set the login option to mixed or just SAML

**Review by Vmihajlovic — APPROVED** [review]
left few comments which are not necessarily blockers in any way. Just some additional clarifications that we might want to add

**Comment by mostafa** [comment]
Thanks @Vmihajlovic for the review! I've made some changes and applied your suggestions.

**Pull request merged**